### ISSUE TITLE
Initiate phase 3 - 508 compliance

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#807]
Need to send program managers list of docs in phase 3 and ask them to decide if they will correct all of them or remove any of them.

**Comment by Maroyafaied** [comment]
Notes to myself

- Reassign reporter letter to RRM that RV identified as not theirs
- Email Joe Coleman, Michael Marchetti, Christina Mathers and Rob Winter
- Look for issue from RV on docs they want to remove

**Comment by Maroyafaied** [comment]
Email Bob Kronebush and Bill Blackburn
